apps/evm/src/pages/Account/NewPage/PerformanceChart/fakeData.ts

```diff
@@ -0,0 +1,602 @@
+export const data = [
+  {
+    apyPercentage: 463.22072070832434,
+    timestampMs: 1750752158000,
```

Comment:
Without formatting, the stored timestamps from the subgraph are in microseconds. Meaning they would have to be either converted to miliseconds, or change the expectation here to microseconds. This can be done by the backend before sending out the response, or it can be sent as is to the frontend.

Reply:
The `/market/history` endpoint currently returns microseconds too if I'm not mistaken, so I'd say let's keep things that way and I'll update the frontend instead.

Reply:
Let's start with this one in miliseconds then, it's basically just removing the least significant digits. And keep here as it already is.